CMakeLists.txt

```diff
@@ -172,6 +176,7 @@ if(${BUILD_GRPC})
       cartographer/cloud/map_builder_server_main.cc
   )
   target_link_libraries(cartographer_grpc_server PUBLIC grpc++)
+  target_link_libraries(cartographer_grpc_server PUBLIC async_grpc)
```

Comment:
After you did find_package above, you should use the variable ASYNC_GRPC_LIBRARIES here (and below)

Reply:
as discussed.